docs/ssss.md

```diff
@@ -1,3 +1,94 @@
 # Simperby Super Simple Summary
 
-TODO
+## DAO
+
+A DAO (Decentralized Autonomous Organization) is an organization that runs on a
+blockchain, governed by rules set in a contract, and operates autonomously
+without any government backing.
+
+For more information on DAOs, refer to [the introduction to DAO
+(WIP)](./dao.md).
+
+## How to Implement a DAO
+
+The most common way to implement a DAO is by **using a mainnet that supports
+smart contracts**. You can code your governance rules in a contract and deploy
+it on the mainnet, which will interact with the mainnet's ecosystem. However,
+this approach **has some disadvantages**:
+
+1. It requires payment of a gas fee.
+2. It is dependent on the chosen mainnet's performance, security, reliability,
+   usability, political risks, etc.
+3. It is limited to the ecosystem of the chosen mainnet and cannot easily
+   interact with other mainnets.
+
+Simperby solves these problems by providing a standalone blockchain for a DAO.
+
+For more details, refer to [the comparisons (WIP)](./comparison.md)
+
+## Simperby
+
+Simperby is a blockchain engine that builds a standalone blockchain for a DAO.
+
+- Trustless
+- Distributed
+- Fault Tolerant
+- Lightweight
+- Highly Interoperable
+- Native Support for Multichain Treasuries
+
+For details on the core protocol, refer to the [protocol
+overview](./protocol_overview.md).
+
+### A Byzantine Fault Tolerant System
+
+- **Builds an independent layer-1 mainnet** like [Cosmos
+  SDK](https://v1.cosmos.network/sdk) or [Substrate](https://substrate.io/).
+- **One mainnet for one DAO**.
+- Has own **BFT consensus** called [Vetomint](./vetomint.md) optimized for
+  sporadic node operations.
+
+### Governance
+
+- A peer-to-peer voting mechanism (governance) is used to make decisions on the
+organization, which is then finalized by the consensus.
+- No tokens, no staking, no mining, no gas fee; Instead, **permissioned and
+explicitly nominated nodes**.
+- Has its own peer-to-peer communication channel for the organization.
+
+### Storage
+
+- Provides a distributed file system for the organization.
+- **Managed as a distributed Git repository**.
+- Blocks and transactions are stored as Git commits.
```

Comment:
This hierarchy seems weird. Consider indenting to
```
 - **Managed as a distributed Git repository**.
     - Blocks and transactions are stored as Git commits.
     - Governance and consensus proposals are stored as Git branches.
```
or rewrite to
```
- - **Managed as a distributed Git repository**: blocks and transactions are stored as Git commits and governance and consensus protocols are stored as Git branches.
```

Reply:
Fixed, and added another bullet point